src/spacegame3/SpaceGame.java

```diff
@@ -80,29 +94,45 @@ public void giveSceneTo(SizableScene scene){
         stage.setScene(scene);
     }
 
+    /*
+        *main method - runs + launches game
+    */
     public static void main(String[] args) {
         launch(args);
     }
 
+    //getter for game started
     public boolean gameStarted() {
         return gameStarted;
     }
 
-
+    /*
+        *sets the planet screen to have landed on the parameter (a CelestialBody object)
+        *returns planet screen
+    */
```

Comment:
use /** */ for methods (just repeating for each one)
this description tells what it does that cant be determine from the method name which is good, maybe just change the end for "on the CelestialBody" or "on the given CelestialBody"